backend/routes/products.py

```
@@ -1,65 +1,53 @@
+    {"id": 12, "name": "Budweiser", "price": 11.99, "image": "https://live.staticflickr.com/1554/25951986653_b85cb79e5d_b.jpg", "category": "beer"},
+    {"id": 13, "name": "Corona Extra", "price": 13.99, "image": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRMiQdEieane-Um0QuXwSSYMsTm9yxZx9JOew&s", "category": "beer"},
+
+    # 🍹 Rum
+    {"id": 14, "name": "Bacardi Rum", "price": 24.99, "image": "https://is.zobj.net/image-server/v1/images?r=sxxY_Ny_lsZ7qSDuIVtFGkAUIF2Mi1pjBEPSENjLeFWq0ynwoYIwHvAdthsw4IoC2O5ERCbGRmR8UvLOu7mwzd4IFqxw_XRDWUbrXELruiVbesnsL6GrmLGD2j1jjwSaJJkWqIX-nzcvia3m6QY7-2XBmGPulD6GFcVdi2EyJZ1kVhPZRbZUNx6RgUCi9tmFKgsU2Nk0L8rb1IpEnTii6tt5MEDg62winr1PDw", "category": "rum"},
+    {"id": 15, "name": "Captain Morgan Rum", "price": 26.99, "image": "https://static.vecteezy.com/system/resources/previews/031/232/601/large_2x/kharkov-ukraine-december-3-2020-captain-morgan-original-spiced-gold-spirit-drink-bottle-on-dark-black-background-elite-alcohol-free-photo.JPG", "category": "rum"},
+    {"id": 16, "name": "Malibu Coconut Rum", "price": 21.99, "image": "https://t3.ftcdn.net/jpg/02/79/97/98/360_F_279979865_JoaUeCMlwLZmrceXe5TtqJXBoUD02TsC.jpg", "category": "rum"},
+
+    # 🍸 Gin
+    {"id": 17, "name": "Tanqueray Gin", "price": 27.99, "image": "https://images.pexels.com/photos/9142626/pexels-photo-9142626.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500", "category": "gin"},
+    {"id": 18, "name": "Bombay Sapphire", "price": 30.99, "image": "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRKdxmu1QZ8knonNMfHZN0o2ie2Htd29VluCw&s", "category": "gin"},
+    {"id": 19, "name": "Hendrick’s Gin", "price": 34.99, "image": "https://as2.ftcdn.net/jpg/03/07/55/41/1000_F_307554176_YgiBDqzYxrMFIVyj73J9BSwBIyBov9CJ.jpg", "category": "gin"},
```

Ruff (0.13.1)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)